### PR TITLE
Add CITATION.cff file for conference

### PR DESCRIPTION
**Pull request by sdruskat**
This PR (finally, sorry folks!) adds a CITATION.cff file for the conference, where authors are all committee members in the order they appear in the `_data/committee/members.yml` file.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,288 @@
+cff-version: 1.2.0
+message: "If you want to cite SORSE, please use the metadata in this file."
+version: "Season 1"
+title: " SORSE - International Series of Online Research Software Event"
```
Comment (sdruskat) [inline CITATION.cff]:
```suggestion
title: " SORSE - International Series of Online Research Software Events"
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    website: https://sdruskat.net
+    orcid: https://orcid.org/0000-0003-4925-7248
+  - family-names: Sommer
+    given-names: Philipp S.
+    affiliation: Helmholtz-Zentrum Geesthacht Centre for Materials and Coastal Research (HZG)
+    city: Geesthacht
+    country: DE
+    website: http://www.philipp-s-sommer.de
+    orcid: https://orcid.org/0000-0001-6171-7716
+    email: philipp.sommer@hzg.de
```
Comment (Chilipp) [inline CITATION.cff]:
```suggestion
    affiliation: Helmholtz-Zentrum Hereon
    city: Geesthacht
    country: DE
    website: https://www.philipp-s-sommer.de
    orcid: https://orcid.org/0000-0001-6171-7716
    email: philipp.sommer@hereon.de
```

**Review by Chilipp — APPROVED** [review]
LGTM! Thanks @sdruskat :smiley:

**Comment by Chilipp** [comment]
feel free to merge this when you are done with it

**Comment by florianthiery** [comment]
thx @sdruskat :-)

**Pull request merged**